### PR TITLE
Adjust Decentraland populate script to work on ropsten testnet

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#296 

### Types of changes

Bug fix (non-breaking change that fixes an issue)

### CODE REVIEW

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js [hunk 1]
```diff
@@ -182,12 +182,20 @@ const createParcel = async parcel => {
...
   const parcelId = await new Promise((resolve, reject) => {
-    action.once("confirmation", async message => {
+    action.on("confirmation", async message => {
       const id = await landContract.encodeTokenId(`${x}`, `${y}`);
+      action.unsubscribe();
       resolve(id);
     });
 
+    action.on("error", message => {
+      action.unsubscribe();
```
Comment (pcowgill) [inline packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Maybe log something here too

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js [hunk 2]
```diff
@@ -224,14 +232,32 @@ const createEstate = async estate => {
...
       resolve(args._estateId);
     });
 
+    // Some error (orphan block, failed tx) events are been trigger only from the confirmationListener
+    // See more: https://github.com/tasitlabs/TasitSDK/issues/253
+    action.on("confirmation", () => {});
+
+    action.on("error", message => {
+      estateContract.unsubscribe();
+      action.unsubscribe();
```
Comment (pcowgill) [inline packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Maybe log something here too

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js [hunk 3]
```diff
@@ -224,14 +232,32 @@ const createEstate = async estate => {
...
   const estateId = await new Promise((resolve, reject) => {
-    estateContract.once("CreateEstate", message => {
+    estateContract.on("CreateEstate", message => {
       const { data } = message;
       const { args } = data;
+      estateContract.unsubscribe();
       resolve(args._estateId);
     });
 
+    // Some error (orphan block, failed tx) events are been trigger only from the confirmationListener
```
Comment (pcowgill) [inline packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]:
```suggestion
    // Some error (orphan block, failed tx) events are being triggered only from the confirmationListener
```

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js [hunk 4]
```diff
@@ -224,14 +232,32 @@ const createEstate = async estate => {
...
+
+    action.on("error", message => {
+      estateContract.unsubscribe();
+      action.unsubscribe();
+      reject();
+    });
+
+    estateContract.on("error", message => {
+      estateContract.unsubscribe();
+      action.unsubscribe();
```
Comment (pcowgill) [inline packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Maybe log something here too

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js [hunk 5]
```diff
@@ -224,14 +232,32 @@ const createEstate = async estate => {
...
+      reject();
+    });
+
     setTimeout(() => {
       estateContract.unsubscribe();
+      action.unsubscribe();
```
Comment (pcowgill) [inline packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Maybe log something here too

**Comment by pcowgill** [comment]
@marcelomorgado Unless you prefer otherwise, I'll assume adding the log messages will gate merging since it's easy enough to introduce them now

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js (hunk 1 above)
Comment (marcelomorgado) [reply packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Done.

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js (hunk 2 above)
Comment (marcelomorgado) [reply packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Done.

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js (hunk 4 above)
Comment (marcelomorgado) [reply packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Done.

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js (hunk 5 above)
Comment (marcelomorgado) [reply packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Done.

**Comment by marcelomorgado** [comment]
> @marcelomorgado Unless you prefer otherwise, I'll assume adding the log messages will gate merging since it's easy enough to introduce them now

Yes, I agree and I've done this.

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js (hunk 1 above)
Comment (pcowgill) [reply packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Thanks! Resolving

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js (hunk 4 above)
Comment (pcowgill) [reply packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Thanks! Resolving

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js (hunk 5 above)
Comment (pcowgill) [reply packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Thanks! Resolving

File: packages/tasit-contracts/src/scripts/populateDecentralandContracts.js (hunk 2 above)
Comment (pcowgill) [reply packages/tasit-contracts/src/scripts/populateDecentralandContracts.js]: Thanks! Resolving

**Pull request merged**